pkg/controller/util.go

```diff
@@ -1087,16 +1087,22 @@ func getDefaultAccessModes(c client.Client, storageClass *storagev1.StorageClass
 	return nil, errors.Errorf("no accessMode defined on StorageProfile for %s StorageClass", storageClass.Name)
 }
 
-func volumeSize(c client.Client, storage *cdiv1.StorageSpec, volumeMode *v1.PersistentVolumeMode) (*resource.Quantity, error) {
+func volumeSize(c client.Client, spec cdiv1.DataVolumeSpec, volumeMode *v1.PersistentVolumeMode) (*resource.Quantity, error) {
```

Comment:
just making sure - is this function is only being called in the context of having storage API?

Reply:
Exactly, as far as I know, It's only called when using the storage API.

Reply:
yes, I suppose now the name is too generic, previously it was obvious from arguments `cdiv1.StorageSpec`.

Reply:
and I supposer this function can be now divided into two parts, 
1. one called from renderPVC from storage API context, 
2. one that actually does calculate the new size (inside the `	if resolveVolumeMode(volumeMode) == v1.PersistentVolumeFilesystem {` )

And the second one can then be reused inside the function  `inflateSizeWithOverhead` in dv controller